src/client/Client.cc

```diff
@@ -9767,6 +9767,8 @@ void Client::getcwd(string& dir, const UserPerm& perms)
 int Client::statfs(const char *path, struct statvfs *stbuf,
 		   const UserPerm& perms)
 {
+  unsigned long int total_files_on_fs = get_root()->rstat.rfiles +
```

Comment:
Should directories be counted as files and added to the count of total number of files on the FS? I''ve done so for now.

Reply:
Yes